clarity/contracts/arkadiko-dao.clar

```diff
@@ -416,11 +416,56 @@
     { name: "liquidator" }
     {
       address: 'ST1PQHQKV0RJXZFY1DGX8MNSNYVE3VGZJSRTPGZGM,
-      qualified-name: 'ST1PQHQKV0RJXZFY1DGX8MNSNYVE3VGZJSRTPGZGM.arkadiko-liquidator-v2-1
+      qualified-name: 'ST1PQHQKV0RJXZFY1DGX8MNSNYVE3VGZJSRTPGZGM.arkadiko-auction-engine-v4-1
```

Comment:
Q: why has the liquidator become the auction engine?

Also I don't see the `liquidate` function called anywhere on `arkadiko-freddie-v1-1` anymore, which is necessary to tag the vault as liquidated and for the user to eventually withdraw leftover collateral after the auction

Reply:
The auction-manager will:
1) start the auction and call `liquidate` on `arkadiko-freddie-v1-1`
2) burn USDA and get the some of the collateral to deposit to `arkadiko-liquidation-rewards-v1-1`
3) finish the auction by calling `finalize-liquidation` on `arkadiko-freddie-v1-1`

No need to have a separate liquidator imo. All liquidation logic is now part of `arkadiko-auction-engine-v4-4`